src/functionalTest/java/uk/gov/hmcts/darts/usermanagement/UserManagementFunctionalTest.java

```diff
@@ -224,4 +228,94 @@ void shouldGetSecurityGroups() throws JsonProcessingException {
                                                                                               new TypeReference<List<SecurityGroupWithIdAndRole>>(){});
         assertFalse(securityGroupWithIdAndRoles.isEmpty());
     }
+
+    @Test
+    void shouldPatchSecurityGroups() throws JsonProcessingException {
```

Comment:
Wondering, should this test be put under the `SecurityGroupFunctionalTest` where you've been previously?
Maybe the test above can be delete just as it also appears in the other place?